### PR TITLE
feat: Add BOB, Mantle, Taiko

### PR DESCRIPTION
**Pull request by avious00**
#### Other notes

Corresponding issue, fixes: https://github.com/hyperlane-xyz/issues/issues/1289

### CODE REVIEW

File: chains/bob/logo.svg
Comment (jmrossy) [inline chains/bob/logo.svg:1]: I can see from the SVG code this isn't minified. Bonus points for running the logos through https://jakearchibald.github.io/svgomg/ before adding here

File: chains/mantle/logo.svg
Comment (jmrossy) [inline chains/mantle/logo.svg:1]: Ditto for this one but also the empty space around the image should be trimmed.
https://jsfiddle.net/knLbjc9a/4/

**Review by jmrossy — COMMENTED** [review]
Can you confirm none of these chains are scrapped? If they are, please add the scrapped ones to the CoreChains enum in /src/core/

File: chains/mantle/logo.svg
Comment (paulbalaji) [inline chains/mantle/logo.svg:1]: still need to trim empty space

Comment (paulbalaji) [reply chains/mantle/logo.svg:1]: couldn't get this to work, the output look fine in jsfiddle but is garbled in the svg viewer
image

**Comment by avious00** [comment]
> Can you confirm none of these chains are scrapped? If they are, please add the scrapped ones to the CoreChains enum in /src/core/

hey @jmrossy could you confirm this more for my future reference? if these are AW-run and added to scraper, then i should also add to /src/core/chain.ts

**Comment by jmrossy** [comment]
> hey @jmrossy could you confirm this more for my future reference? if these are AW-run and added to scraper, then i should also add to /src/core/chain.ts

@avious00 correct. Until https://github.com/hyperlane-xyz/hyperlane-explorer/issues/74 is done, that's required

**Comment by jmrossy** [comment]
@paulbalaji @avious00 Also need a changeset here please

File: chains/mantle/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+blockExplorers:
+  - apiUrl: https://explorer.mantle.xyz/api
+    family: blockscout
+    name: Mantle Mainnet Explorer
+    url: https://explorer.mantle.xyz/
```
Comment (jmrossy) [inline chains/mantle/metadata.yaml:6]: Nit: remove trailing slash

**Review by jmrossy — APPROVED** [review]
LGTM

**Pull request merged**